src/Benchmarks/RabbitMQ/MassTransit/MassTransitTestClient/MassTransitTestClient/Program.cs

```diff
@@ -1,26 +1,21 @@
-﻿using MassTransit;
-using MassTransitTestCommon;
-using System;
+﻿using System;
 using System.Configuration;
 using System.Diagnostics;
 using System.Threading.Tasks;
+using MassTransit;
+using MassTransitTestCommon;
 
 namespace MassTransitTestClient
 {
-    class Program
+    internal class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
+            var taskCount = int.Parse(ConfigurationManager.AppSettings["NoOfThreads"]);
+
             var busControl = Bus.Factory.CreateUsingRabbitMq(cfg =>
             {
-                //Commented out settings below had no effect on client
-                //cfg.AutoDelete = true;
-                //cfg.Durable = false;
-                //cfg.SetQueueArgument("durable", false);
-
-                //Also there seems to be no way to turn off publisher confirms.
-
-                cfg.PrefetchCount = 50;
+                cfg.PrefetchCount = 100;
```

Comment:
I think that's fine, although if you have 100 client threads they'll need 100 consumer threads with an equivalent prefetch count. Otherwise messages get queued in memory. But my testing isn't extensive on non-durable queues.